docs/set-up.asciidoc

```diff
@@ -35,36 +36,284 @@ include::./restify.asciidoc[]
 
 include::./fastify.asciidoc[]
 
+include::./typescript.asciidoc[]
+
 include::./custom-stack.asciidoc[]
 
 include::./lambda.asciidoc[]
 
 [[starting-the-agent]]
 === Starting the agent
 
-IMPORTANT: The Elastic APM agent for Node.js needs to be started as the first thing in your application - *before any other module is required/imported*.
-This means that you should probably require and start it in your applications main file (usually `index.js`, `server.js`, or `app.js`).
+There are a few ways to start the Node.js APM agent. Choose the one that works best for you. The most important considerations for selecting a method are:
+
+- ensuring the APM agent starts early enough, and
+- having a convenient way to configure the agent.
+
+For the Node.js APM agent to be able to fully function, it *must be started before `require(...)` statements for other modules*. The APM agent automatically instruments modules by interposing itself in the import process. If a given module is imported before the APM agent has started, then it won't be able to instrument that module.
+
+
+==== Start methods
+
+[[start-option-require-and-start]]
+===== `require('elastic-apm-node').start(...)`
+
+The most common way to start the APM agent is to require the `elastic-apm-node` module and call the <<apm-start,`start`>> function at the top of your main module. This allows you to use any of the methods to <<configuring-the-agent,configure the agent>>.
```

Comment:
How do you feel about updating this to `.start()`? This would (1) match [this line](https://github.com/elastic/apm-agent-nodejs/pull/2837/files#diff-23427ac409c61f7de0225443c0c7c77aca90b732e73957bad73edb357c30addeR10), and (2) provide just slightly more information to the user to hopefully prevent them from clicking away (unless that's what they actually want to do). Also, does function vs method matter? We use method above and function here.

```suggestion
The most common way to start the APM agent is to require the `elastic-apm-node` module and call the <<apm-start,`.start()`>> function at the top of your main module. This allows you to use any of the methods to <<configuring-the-agent,configure the agent>>.
```

Reply:
Done in commit d410b09a and commit 0e305c1c.

It is a method (in that it is a function that is tied to a "class", inasmuch as JS has classes). Thanks.